shared/cfg/guest-hw.cfg

```diff
@@ -163,7 +163,6 @@ variants:
         # available, it will also try to get first available pf.
         device_name = eth1
         nettype = bridge
-        pf_filter_re = "82576 Gigabit Network"
```

Comment:
How about this, I comment this line instead of removing it.  If someone wants to run other sriov tests, he could uncomment it and reconfigure it as he wants. After all, this param needs to be reconfigured according to the kinds of pf.  Thanks!